x/bank/src/params.rs

```diff
@@ -33,18 +47,17 @@ impl ParamsSerialize for BankParams {
     }
 
     fn to_raw(&self) -> Vec<(&'static str, Vec<u8>)> {
-        let mut hash_map = Vec::with_capacity(2);
-
-        hash_map.push((
-            KEY_DEFAULT_SEND_ENABLED,
-            self.default_send_enabled.to_string().into_bytes(),
-        ));
-
-        // TODO: The send_enabled field is hard coded to the empty list for now
-        // TODO: if params are missing in the cosmos SDK (from genesis at least) then they are set to "null" i.e. [110, 117, 108, 108] when stored
-        hash_map.push((KEY_SEND_ENABLED, "[]".as_bytes().to_vec()));
-
-        hash_map
+        vec![
+            (
+                KEY_DEFAULT_SEND_ENABLED,
+                self.default_send_enabled.to_string().into_bytes(),
+            ),
+            (
+                KEY_SEND_ENABLED,
+                serde_json::to_vec(&self.send_enabled)
```

Comment:
We should keep one of the TODOs because it's still true, maybe with some rewording:
```TODO: if params are missing in the cosmos SDK (e.g the send_enabled field is missing from the genesis json file) then they are set to "null" i.e. [110, 117, 108, 108] when stored. This is different to the Gears behaviour which would fail to parse if e.g. the send enabled field is missing
```

Reply:
Added